scripts/zeus.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -eux
+
+yarn global add @zeus-ci/cli
+
+zeus upload -t "application/zip+aar" ./*/build/outputs/aar/*release.aar
+zeus upload -t "application/zip+jar" ./*/build/libs/*release.jar
```

Comment:
what's about poms? eg
`build/publications/release/pom-default.xml`
or even `*javadocs.jar`, `*sources.jar`?

Reply:
The jars are all match the pattern. The pom is not because their names are all the same. I mentioned on slack. Wither we rename or find another way

Reply:
true about the pom files, but not for sources and javadocs files as they have a `*-sources.jar` and `*-javadocs.jar` pattern if I recall it well.